docs/localization.md

```diff
@@ -13,22 +13,30 @@ If you want to add support for another language you will need to add it to the
 
 ## Translation Process
 
-1. **Extract Strings**
-   Run `npm run make-pot` to get the text out of the source
-   files. This will create a `*.pot` file for each module, as well as a bundle
+### Extract and Import
+
+#### 1 - Extract Strings:
```

Comment:
Can we follow the convention from the readme file? 

```suggestion
#### Step 1: Extract Strings
```

Reply:
Sure, I'll update it with this format.

Reply:
Suggestion applied in https://github.com/Automattic/studio/pull/381/commits/a950e29208704f831f69cdd0fb48b4724bf28df6.